rules_service/settings/base.py

```diff
@@ -29,6 +29,11 @@
     '.app.cloud.gov',
 ]
 
+CORS_ORIGIN_WHITELIST = [
+    'localhost:3000',
+    '127.0.0.1:3000',
+    'https://dsnap-registration.app.cloud.gov',
```

Comment:
I now realize that with us hosting the registration app, we need to create additional environments for it like `staging`, `prod`, etc. Should we use `CORS_ORIGIN_REGEX_WHITELIST` instead?

Reply:
That makes sense! We won't need a regex for the local env though, so should it be a separate array that gets concated with the existing whitelist?

Reply:
The docs are not very clear, but it looks like you can have both: https://github.com/ottoyiu/django-cors-headers/blob/79fec3b325c9cf6bd42923b0b7fefc0df844ef2e/corsheaders/checks.py#L74

Reply:
```
CORS_ORIGIN_REGEX_WHITELIST = [
    'localhost:3000',
    r'127\.0\.0\.1:3000',
    r'^https?://dsnap-registration.*\.app\.cloud\.gov$',
]
```
should work.